main.php

```diff
@@ -117,6 +117,8 @@
 require_once('core/startup/ErrorControlChain.php');
 require_once('core/startup/ParameterConfirmationToken.php');
 
+ini_set('display_errors', 0);
```

Comment:
Is this meant to be here? Obviously this should be off in prod, but do we always want it off?